babel.config.js

```diff
@@ -17,5 +19,6 @@ module.exports = (api) => {
   return {
     presets,
     plugins,
+    ignore: [(filename) => filename === path.resolve(path.join(__dirname, 'packages', 'terra-polyfill', 'src', 'index.js'))],
```

Comment:
With the fix i checked in earlier i think you could switch back to a simpler ignore.
```suggestion
    ignore: ['./packages/terra-polyfill/src/index.js'],
```